go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/letsencrypt/ct-woodpecker
 
 require (
+	github.com/go-sql-driver/mysql v1.4.1
```

Comment:
I think there should also be a removal from `go.mod` (and the accompanying `go.sum`, `modules.txt`, and `vendor` dir) for the sqlite3 driver we aren't using anymore, no?